tests/datavolume_test.go

```diff
@@ -59,8 +59,7 @@ var _ = Describe("DataVolume Integration", func() {
 	})
 
 	runVMIAndExpectLaunch := func(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
-		By("Checking that the DataVolume has succeeded")
-		tests.WaitForSuccessfulDataVolumeImport(vmi, timeout)
+		tests.WaitForSuccessfulDataVolumeImportOfVMI(vmi, timeout)
```

Comment:
Just another nit here, not a real action item: The reader does not know directly what measure the timeout value is using. We might either use a dedicated method name to point this out (`WaitSecondsForSuccessfulDataVolumeImportOfVMI`) or maybe sth in the means of [https://golang.org/pkg/time/#ParseDuration](time.ParseDuration), i.e. "5m" to increase readability. Or we agree that we **always** use seconds.